### ISSUE TITLE
theme not work with hassio 0.117

### ISSUE THREAD

**Issue by riddik14** [basnijholt/lovelace-ios-themes#18]
theme not work with hassio 0.117

**Comment by basnijholt** [comment]
Could you be a bit more elaborate? It does work for me.

**Comment by riddik14** [comment]
in setup panel, theme not work . work in card

**Comment by basnijholt** [comment]
Could you post a screenshot?

**Comment by riddik14** [comment]
![image](https://user-images.githubusercontent.com/37450907/97550156-9561b580-19d1-11eb-90f0-dfa3d1ec3aac.jpeg)

**Comment by riddik14** [comment]
![image](https://user-images.githubusercontent.com/37450907/97550187-a4486800-19d1-11eb-886d-415dbf1112f9.jpeg)

**Comment by basnijholt** [comment]
That is because you are setting the theme for a card!

You should set it by clicking on your initials/profile picture:
<img width="1440" alt="Screenshot 2020-10-29 at 10 31 24" src="https://user-images.githubusercontent.com/6897215/97550463-fdb09700-19d1-11eb-8664-33f796bd6953.png">

**Comment by riddik14** [comment]
No this work but for all panel. I use theme specific for panel and for card. For card and panel non work for me with 0.117. I have clean cache, nothing. This problem on Google chrome and ha app iOS

**Comment by basnijholt** [comment]
And it worked before?

**Comment by riddik14** [comment]
with 0.116.4 it worked as a panel. I only use theme in panel and tab but in panel it doesn't work

**Comment by basnijholt** [comment]
And do other custom themes work? Or is the problem only for this theme?

**Comment by riddik14** [comment]
In card work but wallpaper not work

**Comment by riddik14** [comment]
La trasparenza nemmeno va

**Comment by deltabert** [comment]
Hi Bas, I have the same problem. I used to give every menu a different Theme (color scheme). But now only the Theme selected by the user profile is used throughout all the menu's
![image](https://user-images.githubusercontent.com/5196699/97597823-8e08cf00-1a06-11eb-8290-49eae1b9d396.png)

**Comment by drjjr2** [comment]
Can confirm. I was able to set themes on a per-panel basis and with 0.117, I cannot. It does work if I set it "system-wide" with my user account but then what's the point of being able to specify them for each panel?

**Comment by basnijholt** [comment]
OK great, so this is an issue. However, can anyone check whether this is an issue specific to `lovelace-ios-themes` or whether this is a problem for all custom themes?

**Comment by bcmitri** [comment]
From what I can tell, this is not an issue for all custom themes.

**Comment by basnijholt** [comment]
@bcmitri, did you actually try it?

**Comment by bcmitri** [comment]
Yes.  I didn't try all custom themes, but I tried one other that includes a background and it wasn't displaying the same issue.

**Comment by drjjr2** [comment]
@basnijholt I just tried JuanMTech/google_dark_theme and it also does not change the background except when I specify it as the theme for the user.

I don't think this is specific to your themes.

**Comment by basnijholt** [comment]
@bcmitri, could you please tell me which theme you tried with a custom background and which worked?

I just tried to install https://github.com/Kibibit/hass-kibibit-theme (a theme with a custom background) and it has the same problem as described here. Therefore, I think this problem is not a problem that is specific to this theme but this is a bug that should be reported in https://github.com/home-assistant/frontend.

**Comment by bcmitri** [comment]
I tried the Caule Black Purple Glass theme from the Caule Themes Pack 1 - by caulecriativo.com.

**Comment by drjjr2** [comment]
@basnijholt I have opened issue #7549

**Comment by basnijholt** [comment]
https://github.com/home-assistant/frontend/issues/7549

**Comment by bcmitri** [comment]
Oh, okay.  If this applies specifically to themes that are applied to individual views, that is not what i tried in my test.

Trying that with a different theme, I do indeed see the same results as what are described here.

**Comment by basnijholt** [comment]
@bcmitri, I have tried the theme you mentioned ("Caule Black Purple Glass"), however, I cannot reproduce the behavior you describe. I just see the same problem occur there. Are you 100% sure you tried it with HA version 0.117?

**Comment by bcmitri** [comment]
See my previous comment.
